lib/oven.py

```diff
@@ -346,6 +353,7 @@ def reset(self):
         self.heat_rate_temps = []
         self.pid = PID(ki=config.pid_ki, kd=config.pid_kd, kp=config.pid_kp)
         self.catching_up = False
+        self.emergency_trigger()
```

Comment:
I don't understand why an emergency is triggered here. I think you are using this to shut down power to the kiln relays after each heating cycle.... which seems odd to me.

I would think you would want to shut power to the kiln down when there is a known problem, not 20,000 times during a firing.

Maybe I'm misunderstanding the change?

Reply:
Most everything else looks fine. I would like to see this as optional... so it needs to be tested with and without the setting existing in config.py. It should not error out if the setting does not exist.